regression/testing_tool.py

```diff
@@ -34,6 +35,19 @@
 SUPPORTED_TEST_MODES = ["CORE", "FUTURE", "THOROUGH", "KNOWNBUG", "ALL"]
 FAIL_MODES = ["KNOWNBUG"]
 
+# ESBMC output logs
+# If the user wants to keep the logs, please add "--retain-logs" argument.
+# The logs will be kept in the director named as "20220211_073815_<test_name>".
+NOW = datetime.now().strftime("%Y%m%d_%H%M%S")
+LOG_DIR = ""
+LOG_RETENTION = False
+
+# Use old C and C++ frontend
+OLD_FRONTEND = False
+
+# user defined operation mode directory
+OP_MODE_DIR = ""
```

Comment:
Are you sure all those variables need to be global? Can't you refactor into another class? 

Also this could be a PR by itself

Reply:
Agreed. I'll split this PR into multiple small PRs.